book/plugins/how-to-guides/maximize-compatibility.md

```diff
@@ -18,13 +18,163 @@ Some guidance on each of these approaches:
    - Your install instructions can indicate that a user should install whichever distribution you depend on (e.g., `tiny`, `amplicon`, or `metagenome`) and then illustrate how to install your plugin(s) in that environment however it makes sense (e.g., `conda` or `pip`). Complete install instructions are drafted for you in the `README.md` of plugins that you build using our template (see [](plugin-from-template)).
    - Alternatively, you can compose and share your own distribution of plugins (e.g., building from the `tiny` distribution) that captures the set of functionality you’d like to share.
 
-Either of these approaches is totally fine.
-The former is an easier starting point.
+Either of these approaches is totally fine, with the following caveat.
+The former is an easier starting point, and will allow us to provide more troubleshooting assistance for any installation issues that users may encounter. While the latter provides you with more flexibility in your environment construction, our assistance with any install issues or environment conflicts that users may run into will be more limited (and will ultimately be your responsibility to troubleshoot and resolve).
```

Comment:
It's a bit of an odd convention, but we write this with one sentence per line. That just makes diffs easier to view, and the rendering engine ignores the single line breaks. Could you break this up so it's one sentence per line?

Reply:
Oh yeah, can do!